utils/ssh.py

```diff
@@ -16,6 +16,7 @@
 from utils.timeutil import parsetime
 
 
+paramiko.util.log_to_file('log/ssh.log')
```

Comment:
This can fail with unexpected ways due to relative paths. To do this correctly, you'll probably want to import `log_path` from `utils.path`, and pass `log_path.join('ssh.log').strpath` as the argument here.